PhysicsTools/PatAlgos/plugins/PATLostTracks.cc

```diff
@@ -255,16 +273,100 @@ void pat::PATLostTracks::addPackedCandidate(std::vector<pat::PackedCandidate>& c
   cands.emplace_back(
       pat::PackedCandidate(p4, trk->vertex(), trk->pt(), trk->eta(), trk->phi(), id, pvSlimmedColl, pvSlimmed.key()));
 
-  if (trk->quality(reco::TrackBase::highPurity))
-    cands.back().setTrackHighPurity(true);
-  else
-    cands.back().setTrackHighPurity(false);
+  cands.back().setTrackHighPurity(trk->quality(reco::TrackBase::highPurity));
 
-  if (trk->pt() > minPtToStoreProps_ || trkStatus == TrkStatus::VTX)
-    cands.back().setTrackProperties(*trk, covarianceSchema_, covarianceVersion_);
-  if (pvOrig.trackWeight(trk) > 0.5) {
-    cands.back().setAssociationQuality(pat::PackedCandidate::UsedInFitTight);
+  if (trk->pt() > minPtToStoreProps_ || trkStatus == TrkStatus::VTX) {
+    if (std::abs(id) == 11 || trkStatus == TrkStatus::VTX) {
```

Comment:
To recover old behavior I planned to set all items in the schema vector to 0 which will have the same result, but I agree your solution is a bit better as it forces same packed schema for all cases. However, it is not granted by `useLegacySetup_` flag that it will be legacy schema, i.e. 0, as schema depends on other configuration argument (which is by coincidence indeed 0).